TEST_APPS/device/nfcapp/nfccommands.cpp

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright (c) 2018 ARM Limited. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ * Licensed under the Apache License, Version 2.0 (the License); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <string>
+#include <vector>
+#include <stdlib.h>
+#include "mbed.h"
+#include "mbed_events.h"
+#include "mbed-client-cli/ns_cmdline.h"
+#include "rtos\Thread.h"
+
+#include "nfctestshim.h"
+#include "nfcprocess.h"
+#include "nfccommands.h"
+
+events::EventQueue nfcQueue;
```

Comment:
are these meant to be global?
you also mix naming convetions, camel, snake (and hungarian notation?)

Reply:
Coding style 
https://os.mbed.com/docs/v5.11/contributing/style.html